plugins/name/name.go

```diff
@@ -0,0 +1,40 @@
+package Name
+
+import (
+  "regexp"
+  "math/rand"
+  "time"
+
+  "github.com/tucnak/telebot"
+  "github.com/asdine/storm"
+
+  "github.com/focusshifter/muxgoob/registry"
+)
+
+type NamePlugin struct {
+}
+
+var db *storm.DB
+var rng *rand.Rand
+
+func init() {
+  registry.RegisterPlugin(&NamePlugin{})
+}
+
+func (p *NamePlugin) Start(sharedDb *storm.DB) {
+  db = sharedDb
+  rng = rand.New(rand.NewSource(time.Now().UnixNano()))
+}
+
+func (p *NamePlugin) Process(message telebot.Message) {
+	bot := registry.Bot
+	rngInt := rng.Int()
+
+	hohlyExp := regexp.MustCompile(`yuriyglebov|dbobylev.*$`)
```

Comment:
Rename to `usernameExp` and move usernames to the configuration. Also that regexp is wrong and matches not just `username`, but `anythingusernameanything`.